great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -624,14 +624,18 @@ def construct_versioned_url(
         """Construct the correct url for a given resource."""
         version = cls._ENDPOINT_VERSION_LOOKUP.get(resource_name, EndpointVersion.V0)
 
+        # urljoin strips the last component of the path in base_url if it doesn't end
+        # with a / which we don't want so we add a / to the end if one isn't present
+        b = base_url if base_url[-1] == "/" else base_url + "/"
```

Comment:
Non-blocking: should we be sanitizing this at the config layer?

Also even though it's a short-lived variable, can we give it a descriptive name? `sanitized_base_url` or something?

Reply:
I've moved this sanitization into the `__init__` of `GXCloudConfig`.

Reply:
Nice! thanks!